File: find difference between sum of diagonals/find difference between sum of diagonals.cpp
#include <iostream>

using namespace std;

int main(){
        int main = 0;
        int matrix[3][3] = {
        {1, 2, 3},
        {4, 5, 6},
        {7, 8, 9}
    };
        int secondary=0;
        int b=Grid.size();

        for (int i=0;i<b;i++){
            main += Grid [i][i];
            secondary +=Grid[i][b-i-1];
        }
            
        int difference = abs(main - secondary);
        cout<<difference;
    
}

File: find difference between sum of diagonals/find difference between sum of diagonals.py
main = 0
Grid = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
]
secondary=0
b=len(Grid)
for i in range(b):
    main += Grid[i][i]
    secondary+= Grid[i][b-1-i]
difference = abs(main - secondary)
print(difference)